### ISSUE TITLE
how to finetuning with roberta-large

### ISSUE THREAD

**Issue by zhaoguangxiang** [huggingface/pytorch-transformers#1239]
## 🐛 Bug

Model I am using (Roberta-large):

Language I am using the model on (English, ):

The problem arise when using:
* [ ] my own modified scripts: (give details)
export CUDA_VISIBLE_DEVICES=2
export GLUE_DIR=/home/zhaoguangxiang/bert/glue_data
DATA=MNLI
NUM_CLASSES=3
LR=1e-5
MAX_SENTENCES=32
TOTAL_NUM_UPDATES=123873
WARMUP_UPDATES=7432
for seed in 42
do
    python3 examples/run_glue.py \
    --model_type bert \
    --model_name_or_path https://s3.amazonaws.com/models.huggingface.co/bert/roberta-large-pytorch_model.bin \
    --task_name ${DATA} \
    --do_train \
    --do_eval \
    --do_lower_case \
    --data_dir $GLUE_DIR/${DATA} \
    --save_steps 10000 \
    --logging_steps 1000 \
    --max_seq_length 512 \
    --max_steps ${TOTAL_NUM_UPDATES}  --warmup_steps ${WARMUP_UPDATES} --learning_rate ${LR} \
    --per_gpu_eval_batch_size 32   \
    --per_gpu_train_batch_size 32   \
    --seed ${seed} \
    --output_dir checkpoint/roberta_${DATA}_output_seed${seed}/
done
The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)

## To Reproduce

1. run scripts
2. i will see 
![image](https://user-images.githubusercontent.com/17742385/64612608-cca88800-d406-11e9-82d8-3e45e02f8577.png)

## Expected behavior

load or download roeberta and s tart  training

## Environment

* PyTorch Transformers version (or branch): latest
* Using GPU ? yes
* Distributed of parallel setup ? no

**Comment by thomwolf** [comment]
What do you mean by "latest" version of PyTorch tranformers.

Are you using a release or installing from source from master?

**Comment by zhaoguangxiang** [comment]
> What do you mean by "latest" version of PyTorch tranformers.
> Are you using a release or installing from source from master?
Thanks for your reply.
pytorch transformers version 1.1.0;
 downloading the code from master;
examples of finetuning roberta were not given in the docs.

**Comment by zhaoguangxiang** [comment]
this is not a bug,   since i write the wrong model_type, that should be 'roberta'
